src/export.js

```diff
@@ -1,5 +1,6 @@
 // For browser, export only select globals
-if ( typeof window !== "undefined" ) {
+if ( window.document && window.HTMLDocument &&
+		window.document instanceof window.HTMLDocument ) {
```

Comment:
this is the most precise check I've got, we might want to simplify if to just check window.document.